bin/bossfetch

```diff
@@ -119,6 +121,26 @@ def main():
         print('No files to fetch.')
         return 0
 
+    if args.globus:
+        try:
+            remote_endpoint, local_endpoint = args.globus.split(':')
+            print('Globus endpoints (remote, local): {0}, {1}'.format(remote_endpoint, local_endpoint))
+        except ValueError as e:
+            print('Error while parsing globus endpoints: {}'.format(str(e)))
+            return -1
+        if args.save:
+            with open(args.save,'w') as f:
+                for remote_path in remote_paths:
+                    local_path = mirror.local_path(remote_path)
+                    f.write('{0}{1} {2}{3}\n'.format(remote_endpoint, remote_path, local_endpoint, local_path))
+            print('Saved {0} globus transfer remote/local file name pairs to {1}.'.format(num_files,args.save))
```

Comment:
The non-globus `if args.save:` block has similar print statement, that's why I didn't include one here.

It actually looks like `--verbose` is used to show the progress bar. I think it's generally okay to print *something* without having the user specify `--verbose`. 

I think the bits of information output here are exactly the pieces the user should check before they decide to initiate the file transfer.